azkaban-web-server/build.gradle

```diff
@@ -12,7 +12,28 @@ buildscript {
 }
 
 plugins {
-    id 'com.craigburke.bower-installer' version '2.5.1'
+  id "com.moowork.node" version "0.13"
+}
+
+node {
```

Comment:
Hey @fsi206914 
Could you please add some comments about the role of this plugin and how's it being used.